groups/chips-rep-db/ec2.tf

```diff
@@ -43,6 +43,26 @@ module "db_ec2_security_group" {
     }
   ]
 
+  ingress_with_source_security_group_id = [for group in local.source_security_group_id :
+    {
+      from_port                = 3872
+      to_port                  = 3872
+      protocol                 = "tcp"
+      description              = "For OEM Adgent"
+      source_security_group_id = group
+    }
+  ]
+
+    ingress_with_source_security_group_id = [for group in local.source_security_group_id :
+    {
+      from_port                = 4903
+      to_port                  = 4903
+      protocol                 = "tcp"
+      description              = "For OEM Adgent"
+      source_security_group_id = group
+    }
+  ]
+
```

Comment:
This will allow access to these ports to all security groups in the profile var list. Is this intended?

It may be better to create a new `aws_security_group_rule` resource that gets attached to the existing security group which, when coupled with a dedicated list and data lookup, could be far more targeted and specific.